renormalizer/mps/mps.py

```diff
@@ -639,31 +644,170 @@ def expand_bond_dimension(self, hint_mpo=None, coef=1e-10, include_ex=True):
         self.compress_config.bond_dim_max_value += self.bond_dims_mean
         return (self + expander.scale(coef, inplace=True)).canonicalise().canonicalise().canonical_normalize()
 
-    def evolve(self, mpo, evolve_dt) -> "Mps":
+    def evolve(self, mpo, evolve_dt, normalize=True) -> "Mps":
 
         method = {
             EvolveMethod.prop_and_compress: self._evolve_prop_and_compress,
+            EvolveMethod.prop_and_compress_tdrk4: self._evolve_prop_and_compress_tdrk4,
+            EvolveMethod.prop_and_compress_tdrk: self._evolve_prop_and_compress_tdrk,
             EvolveMethod.tdvp_mu_vmf: self._evolve_tdvp_mu_vmf,
             EvolveMethod.tdvp_vmf: self._evolve_tdvp_mu_vmf,
             EvolveMethod.tdvp_mu_cmf: self._evolve_tdvp_mu_cmf,
             EvolveMethod.tdvp_ps: self._evolve_tdvp_ps,
             EvolveMethod.tdvp_ps2: self._evolve_tdvp_ps2
         }[self.evolve_config.method]
         new_mps = method(mpo, evolve_dt)
-        if np.iscomplex(evolve_dt):
-            new_mps.normalize(1.0)
+        if normalize:
+            if np.iscomplex(evolve_dt):
+                new_mps.normalize(1.0)
+            else:
+                new_mps.normalize(None)
+        return new_mps
+    
+    def _evolve_prop_and_compress_tdrk4(self, mpo, evolve_dt) -> "Mps":
```

Comment:
Why this should be a separate method? What is the difference between this method and `_evolve_prop_and_compress_tdrk` with `rk_solver == "C_RK4"`?

Reply:
`_evolve_prop_and_compress_tdrk` is more general. 
`_evolve_prop_and_compress_tdrk4` is legecy code.  
Currently, I keep it for test.